### PR TITLE
[SEI] Promote packages to GA

### LINKED ISSUE #8389: [Security Integrations] Moving integrations from Tech Preview to GA
We currently have a number of integrations under Tech Preview, which have been available for some and we can confidently move to GA. 

- [ ] Amazon Security Lake
- [ ] Arista NG Firewall
- [ ] Cisco Nexus
- [x] Custom Azure Blob https://github.com/elastic/integrations/pull/8577
- [x] Custom Google Cloud Storage https://github.com/elastic/integrations/pull/8515
- [ ] Google Security Command Center
- [ ] Imperva
- [ ] Mandiant Advantage
- [ ] Maltiverse
- [ ] Microsoft Defender for Cloud
- [ ] Microsoft Entra ID Entity Analytics
- [ ] Okta Entity Analytics
- [ ] OpenCTI
- [ ] Palo Alto Prisma Cloud
- [ ] Qualys VMDR
- [ ] SentinelOne Cloud Funnel
- [ ] Symantec EDR Cloud
- [ ] Trellix EDR Cloud
- [ ] Wiz

~~Custom AWS Logs (confirm with o11y/cloud team??)~~ (Tracked separately by: https://github.com/elastic/integrations/issues/8919)

### PR DESCRIPTION
**Pull request by taylor-swanson**
## Proposed commit message

- Promote many packages to GA
- Upgraded package-spec to 3.0.0 where necessary

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

elastic-package test

## Related issues

- Closes #8389

### CODE REVIEW

File: packages/aws_logs/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "1.0.0"
+  changes:
+    - description: Release package as GA.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/8833
```
Comment (taylor-swanson) [inline packages/aws_logs/changelog.yml:5]: @elastic/obs-ds-hosted-services, `aws_logs` was included in the list of packages to promote to GA (see https://github.com/elastic/integrations/issues/8389). If there any objections to this and the package is not ready for GA, I'll remove it from this list.

**Comment by taylor-swanson** [comment]
As a side note, elastic-package performed a lot of formatting/whitespace changes on some of the files, so it may help to hide whitespace changes to make it quicker to review.

**Review by kcreddy — APPROVED** [review]
LGTM 👍🏼

**Review by efd6 — APPROVED** [review]

**Comment by taylor-swanson** [comment]
> @taylor-swanson sorry for late notice. Is it possible to keep SentinelOne Cloud Funnel in Tech Preview as @w0rk3r has some mapping adjustments to make to ensure our detection rules can leverage S1 Cloud Funnel data. Easier to make those mapping adjustments while in Tech Preview.

Absolutely, I'll remove it from the list now.

As a note, I don't think the numerous test failures I'm hitting are anything to do with the integrations, there's something weird going on with CI lately.

**Pull request merged**